modules/rosa-sts-account-wide-role-and-policy-commands.adoc

```diff
@@ -86,7 +91,40 @@ aws iam create-policy \
 	--tags Key=rosa_openshift_version,Value=4.8 Key=rosa_role_prefix,Value=ManagedOpenShift Key=operator_namespace,Value=openshift-image-registry Key=operator_name,Value=installer-cloud-credentials
 ----
 
+[discrete]
+== Using `auto` mode for role creation
```

Comment:
Even though this heading will not appear in the TOC, I believe it should have a heading anchor.